yt/utilities/chemical_formulas.py

```diff
@@ -34,12 +34,13 @@ def __repr__(self):
         return self.formula_string
 
 
-def compute_mu():
-    # Assume full ionization and cosmic abundances
-    # This assumes full ionization!
-    muinv = 2.0 * _primordial_mass_fraction["H"] / ChemicalFormula("H").weight
-    muinv += 3.0 * _primordial_mass_fraction["He"] / ChemicalFormula("He").weight
+def compute_mu(ion_state):
+    if ion_state == "ionized" or ion_state is None:
+        Z_H = 2.0
+        Z_He = 3.0
+    elif ion_state == "neutral":
+        Z_H = 1.0
+        Z_He = 2.0
```

Comment:
Shouldn't this be 1.0? With that, I get neutral mu ~ 1.22.

Reply:
@brittonsmith sorry--what do you mean by "this"?

Reply:
@brittonsmith argh--sorry, you're right. I have no idea where my head is at. will fix.